src/widget/OktaSignIn.js

```diff
@@ -45,7 +45,7 @@ var OktaSignIn = (function () {
      * @param opts - OAUTH options to refresh the idToken
      */
     function refreshIdToken(idToken, callback, opts) {
-      authClient.refreshIdToken(idToken, opts).then(callback)
+      authClient.idToken.refresh(opts).then(callback)
```

Comment:
Now that we don't take an idToken param, we have a few options:

1. Change the signature of `refreshIdToken` not to take an idToken. This would be a breaking change and thus, a major version bump.
2. Keep the current signature, but shuffle the arguments if the first argument is a function. We could also warn consumers that passing an idToken is deprecated when one is passed. This wouldn't be a breaking change, but it does make the refresh method a little odd.
3. Keep the current signature, but customers pass `null` if they don't want to provide an idToken.